templates/views/beers.html

```diff
@@ -1,23 +1,88 @@
 {% extends "templates/layouts/default.html" %}
 
+{% block intro %}
+
+<div class="row">
+  <h1 align="center">Beers</h1>
+</div>
+
+{% endblock %}
+
 {% block content %}
-    <div style="background-color:#000;padding:80px 0;">
-        <div class="container">
-            <h1>Beers</h1>
-            <p>which are great</p>
-        </div>
+
+<div class="container-fluid">
+  <div class="row">
+    <h2 align="center">Featured Beers</h2>
+  </div>
+
+  <div class="col-md-12" align="center">
+    {% for featuredBeer in filters.featuredBeers %}
+    {% if loop.index0%2 == 0 %}
+    <div class="col-md-12" align="center">
+      {% endif %}
+      {% if featuredBeer.beerImage.url != nil %}
+      <div class="col-md-6" align="center">
+        <a href="/beerProfile/{{featuredBeer.slug}}">
+          <img src="{{featuredBeer.beerImage.url}}" class="img-thumbnail">
+          <br>
+          {{featuredBeer.beerName}}
+        </a>
+      </div>
+      {% else %}
+      <div class="col-md-6" align="center">
+        <a href="/beerProfile/{{featuredBeer.slug}}">
+          <img src="https://s3.amazonaws.com/upload.uxpin/files/193825/198806/Beer_0000_Layer-Comp-1.png" class="img-thumbnail">
+          <br>
+          {{featuredBeer.beerName}}
+        </a>
+      </div>
+      {% endif %}
+      {% if loop.index0%2 != 0 %}
     </div>
+    {% endif %}
+    {% endfor %}
+  </div>       
 
-    <div style="background-color:#202020;padding:80px 0;">
-        <div class="container">
-            <p>Featured beers</p>
-            <button class="btn btn-primary">Try these beers</button>
-        </div>
+  <div class="row">
+    <h2 align="center">All Beers</h2>
+  </div>
+
+  <div class="col-md-12" align="center">
+    {% for beer in data.allBeers %}
+    {% if loop.index0%6 == 0 %}
+    <div class="col-md-12" align="center">
+      {% endif %}
+        <a href="/beerProfile/{{beer.slug}}">
+        {% if beer.beerImage.url != nil %}
+        <div class="col-md-2" align="center">
+          <img src="{{beer.beerImage.url}}" class="img-thumbnail">
+          <br>
+          {{beer.beerName}}
+        </a>
+      </div>
+      {% else %}
+      <div class="col-md-2" align="center">
+        <a href="/beerProfile/{{beer.slug}}">
+          <img src="https://s3.amazonaws.com/upload.uxpin/files/193825/198806/Beer_0000_Layer-Comp-1.png" class="img-thumbnail">
+          <br>
+          {{beer.beerName}}
+        </a>
+      </div>
+      {% endif %}
+      {% if (loop.index0+1)%6 == 0 %}
     </div>
+    {% endif %}
+    {% endfor %}
+  </div>   
 
-    <div style="background-color:#000;padding:80px 0;">
-        <div class="container">
-            <p>All beers</p>
-        </div>
+  <div class="col-md-12" align="center">
+    <div class="container">
+      <p>Featured beers</p>
+      <button class="btn btn-primary">Try these beers</button>
     </div>
-{% endblock %}
\ No newline at end of file
+  </div>
```

Comment:
Your closing divs should be in the block not outside it. Nothing outside of blocks gets actually put on the page.